app/helpers/redirect_helper.rb

```diff
@@ -12,12 +12,13 @@ def initialize(previously_published_item, previously_drafted_item, payload, call
     def create
       return unless path_has_changed?
 
-      redirect_payload = RedirectPresenter.present(
+      redirect_payload = RedirectPresenter.new(
         base_path: previous_base_path,
         public_updated_at: Time.zone.now,
         redirects: redirects_for(previous_routes, previous_base_path, payload[:base_path]),
-        publishing_app: payload[:publishing_app]
-      ).merge(content_id: SecureRandom.uuid)
+        publishing_app: payload[:publishing_app],
+        content_id: SecureRandom.uuid,
```

Comment:
It'll just be falling back to en so isn't a problem.

However by introducing it into the input of RedirectPresenter and not including it in the output seems a bit prone to a confusing in scenario. I.e. you can now create a "fr" RedirectPresenter and if you pass it to redirect helper it will turn to "en"

Not that redirects need a locale ¯\_(ツ)_/¯